perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/columnProviders/OverviewAttributesColumnProvider.java

```diff
@@ -0,0 +1,199 @@
+		return defaultColumns;
+	}
+
+	@Override
+	public PerunColumnType getDefaultSortColumn() {
+		return PerunColumnType.NAME;
+	}
+
+	@Override
+	public boolean isDefaultSortColumnDescending() {
+		return false;
+	}
+
+	@Override
+	public PerunDataGrid.PerunSelectionEvent<Attribute> getDefaultSelectionEvent() {
+
+		// by default all VOs can be selected
+		return object -> (object != null);
+
+	}
+
+	@Override
+	public PerunDataGrid.PerunFilterEvent<Attribute> getDefaultFilterEvent() {
+		return (columnTypeSet, text, object) -> {
+			if (object == null || text == null) return false;
+
+			if (columnTypeSet == null || columnTypeSet.isEmpty()) {
+				// use default filter - ID,displayName,description
+				columnTypeSet = new HashSet<PerunColumnType>(Arrays.asList(PerunColumnType.NAME, PerunColumnType.DESCRIPTION));
+			}
+			for (PerunColumnType columnType : columnTypeSet) {
+				if (columnType.equals(PerunColumnType.ID) && Integer.toString(object.getId()).contains(text)) {
+					return true;
+				} else if (columnType.equals(PerunColumnType.NAME) && object.getName() != null &&
+								   object.getName().toLowerCase().contains(text.toLowerCase())) {
+					return true;
+				} else if (columnType.equals(PerunColumnType.DESCRIPTION) && object.getDescription() != null &&
+								   object.getDescription().toLowerCase().contains(text.toLowerCase())) {
+					return true;
+				}
+			}
+			return false;
+		};
+	}
+
+	@Override
+	public void addColumnToTable(PerunDataGrid<Attribute> table, PerunColumnType column) {
+		addColumnToTable(table, column, null, 0);
+	}
+
+	@Override
+	public <C> void addColumnToTable(PerunDataGrid<Attribute> table, PerunColumnType column, FieldUpdater<Attribute, C> updater) {
+		addColumnToTable(table, column, updater, 0);
+	}
+
+	@Override
+	public void addColumnToTable(PerunDataGrid<Attribute> table, PerunColumnType column, int widthInPixels) {
+		addColumnToTable(table, column, null, widthInPixels);
+	}
+
+	@Override
+	public <C> void addColumnToTable(final PerunDataGrid<Attribute> table, PerunColumnType column, FieldUpdater<Attribute, C> updater, int widthInPixels) {
+
+		switch (column) {
+			case NAME:
+				PerunColumn<Attribute, String> nameColumn = createColumn(
+						column,
+						Attribute::getName,
+						this.getFieldUpdater(table)
+				);
+
+				nameColumn.setSortable(true);
+				table.getColumnSortHandler().setComparator(nameColumn, new PerunComparator<Attribute>(PerunColumnType.NAME));
+				nameColumn.setColumnType(column);
+				table.addColumn(nameColumn, translation.name());
+				if (widthInPixels > 0) {
+					table.setColumnWidth(nameColumn, widthInPixels + "px");
+				} else {
+					// by default not with fixed width
+					table.setColumnWidth(nameColumn, "30%");
+				}
+				break;
+
+			case ATTR_VALUE:
+				PerunColumn<Attribute, String> attrValueColumn = createColumn(
+						column,
+						attr -> {
+							if (attr.getType().equals("java.util.LinkedHashMap")) {
```

Comment:
Make sure, that you support largeString and largeList as mentione before (please check for correct class name from perun-core sources).

Reply:
Done.